resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -325,6 +351,7 @@ private[spark] object KubernetesUtils extends Logging {
     }
   }
 
+  @Since("3.0.0")
   def buildPodWithServiceAccount(serviceAccount: Option[String], pod: SparkPod): Option[Pod] = {
```

Comment:
This was backported to 2.4.8:
https://github.com/apache/spark/blob/branch-2.4/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala#L66

Reply:
It's not released yet, @attilapiros ! :)

Reply:
2.4.8 RC3 will fail again. Please see this ancient bug fix.
- https://github.com/apache/spark/pull/32404#issuecomment-829894069